plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -90,12 +97,16 @@ func (p *Plugin) InspectFeatureGates(featureGates featuregate.FeatureGate) {
 	p.expansionRecoveryEnabled = featureGates.Enabled(features.RecoverVolumeExpansionFailure)
 	p.dynamicResourceAllocationEnabled = featureGates.Enabled(features.DynamicResourceAllocation)
 	p.allowInsecureKubeletCertificateSigningRequests = featureGates.Enabled(features.AllowInsecureKubeletCertificateSigningRequests)
+	p.serviceAccountNodeAudienceRestriction = featureGates.Enabled(features.ServiceAccountNodeAudienceRestriction)
 }
 
 // SetExternalKubeInformerFactory registers an informer factory into Plugin
 func (p *Plugin) SetExternalKubeInformerFactory(f informers.SharedInformerFactory) {
 	p.podsGetter = f.Core().V1().Pods().Lister()
 	p.nodesGetter = f.Core().V1().Nodes().Lister()
+	p.csiDriverGetter = f.Storage().V1().CSIDrivers().Lister()
+	p.pvcGetter = f.Core().V1().PersistentVolumeClaims().Lister()
+	p.pvGetter = f.Core().V1().PersistentVolumes().Lister()
```

Comment:
is anything else in kube-apiserver watching these already? trying to understand what new informers we're starting

Reply:
I didn't find any other places in kube-apiserver it's already being used

Reply:
Looks like node authorizer is already watching persistent volumes via an informer

CSIDriver objects are expected to be ~few and ~slow-moving, informer for those is fine

Starting a new PVC informer in the apiserver is the most impactful addition here (in terms of memory / CPU impact). Trying to figure out how to weigh the impact of that and if that matters.